### ISSUE TITLE
[6.x] player_levelstats

### ISSUE THREAD

**Issue by funjoker** [TrinityCore/TrinityCore#13823]
Update of player_levelstats Level 1-100.
Clean table.
Only races with playable classes combination.
Also race stat offest.

DB: TDB6

https://gist.github.com/funjoker/e141bc0258ab135e9db0
(Reworks will be applied)

**Comment by streetrat** [comment]
just tested lvl 1 human mage, it has 12 int in db and 24 at blizz
level 90+ stats seem to be right

**Comment by funjoker** [comment]
I'll check that. Thanks for feedback

**Comment by funjoker** [comment]
Int values are wrong yes.
The other values are correct. For Level 1 Mage.

**Comment by funjoker** [comment]
hmm if i compare the INT of several mages at level 90+ it's still different

**Comment by funjoker** [comment]
Revision 2: Updated Inte Value for Human Mages Level 1-21

**Comment by Orphus** [comment]
nice, though ive been checking this out with rat, this int stuff applies for classes(mage and priest and warlock confirmed). Could you possibly check differences for other classes, perhaps rogue can have enhanced agi, war str etc..

**Comment by funjoker** [comment]
I'll check them.
If you have any updates do not hesitate to post them. I'll apply them with Credits to you

**Comment by Orphus** [comment]
I dont have time to lvlup to see where bonus end, though i can post you bonus stats for each class on lvl 1(i have only trial and finals nearby, so i dont have much time) expect them in next post.)

**Comment by Orphus** [comment]
cant test DK, though he starts at 55 so i wouldnt expect any boost. 
bonuses are verified through multiple(2-4) races, so it would be safe to say bonus is per class rather then race/class.
If you implement those bonuses, work on player_levelstats is finished.

on lvl 1 -
Warrior: no boost
Warlock: int +12
Shaman: int +12
Rogue: no boost
Priest: int +12
Druid: int +12
Mage: int +12
Monk: int +12
Hunter: no boost
Paladin: int +12

So looking at posted celestalon datas:
Level Death Knight Druid Hunter Mage Monk Paladin Priest Rogue Shaman Warlock Warrior
1 7 12 10 12 12 12 12 8 12 12 8

if you start with 12 intel, you gain additional 12 :)

**Comment by danlapps** [comment]
I have random toons at random levels, would that help you confirm ?  Like a mage 49, a demo 93, a priest 40.

**Comment by danlapps** [comment]
Base intel stat from live : 
warlock 94 : 682
mage 49 : 106
priest 40 : 82

**Comment by funjoker** [comment]
I need race too.

**Comment by funjoker** [comment]
I thought it to add 12 int too. But wasnt correct.
Humans dont get additional stats. So if i have this datas clear, i am able to calculate other race

**Comment by danlapps** [comment]
human warlock 94 : 682
human  mage 49 : 106
worgen priest 40 : 82

**Comment by funjoker** [comment]
Orphus. It's not that easy. Add 12 Int to them and check the stas with retail. They won't fit.

**Comment by funjoker** [comment]
Example:

dranei shaman 100 : 1162

Celestalon = 1042

+12 is not 1162

**Comment by funjoker** [comment]
inte values changed since celestalon posted them

**Comment by danlapps** [comment]
human pal 100 : 1042
dranei mage 100 : 1162

**Comment by Orphus** [comment]
something boost it, I have draenei mage 100 and without equip i got 1107 int

**Comment by danlapps** [comment]
Maybe some talent ? Or spec bonuses ?? Ill try and check tonight

**Comment by danlapps** [comment]
Try something like Dranei bonus : 10+(lvl*1.1) rounded up

**Comment by DDuarte** [comment]
TL;DR on this? Have the values been confirmed? Good to merge?

**Comment by Orphus** [comment]
most values are correct, initial few lvl int boost needs to be implemented in it...there are some high lvl mismatches, yet I believe, they are caused by some aura

**Comment by msoky** [comment]
mismatches is not big...so this can be merge
